mikeio/dataset/_dataset.py

```diff
@@ -1830,6 +1837,29 @@ def _multiply_value(self, value: float) -> "Dataset":
             validate=False,
         )
 
+    def _divide_value(self, value: float | "Dataset") -> "Dataset":
+        try:
+            if isinstance(value, self.__class__):
+                self._check_datasets_match(value)
+                data = [
+                    self[x].to_numpy() / value[y].to_numpy()
+                    for x, y in zip(self.items, value.items)
+                ]
+            else:
+                data = [self[x].to_numpy() / value for x in self.items]
+        except TypeError:
+            raise TypeError(f"{value} could not be divided to Dataset")
+        items = deepcopy(self.items)
```

Comment:
I guess the unit is often not the same after division 🤔

Reply:
How to handle units in a pragmatic way is what I am struggling with, and I suppose one of the reasons that this is not already supported.